src/mdyml/convert_cisagov.py

```diff
@@ -0,0 +1,211 @@
+"""convert_cisagov markdown conversion tool.
+
+Fetches the contents of the cisagov GitHub repository and converts it to YAML.
+
+EXIT STATUS
+    This utility exits with one of the following values:
+    0   No error occurred.
+    >0  An error occurred.
+
+Usage:
+  convert-cisagov [--log-level=LEVEL]
+  convert-cisagov (-h | --help)
+
+Options:
+  -h --help              Show this message.
+  --log-level=LEVEL      If specified, then the log level will be set to
+                         the specified value.  Valid values are "debug", "info",
+                         "warning", "error", and "critical". [default: info]
+"""
+
+# Standard Python Libraries
+from datetime import datetime, timezone
+import html
+import logging
+import sys
+from typing import Any
+import urllib.request
+
+# Third-Party Libraries
+import dateparser
+import docopt
+import ruamel.yaml
+from schema import And, Schema, SchemaError, Use
+
+from . import DEFAULT_CVE_ID, MD_LINK_RE, ORDERED_CVE_IDS
+from ._version import __version__
+
+RAW_URL = "https://raw.githubusercontent.com/cisagov/log4j-affected-db/develop/SOFTWARE-LIST.md"
+
+EXPECTED_COLUMN_NAMES = [
+    "vendor",
+    "product",
+    "versions",
+    "status",
+    "update_available",  # This column will be ignored as it can be inferred from patched_versions
+    "vendor_link",
+    "notes",
+    "references",
+    "last_updated",
+]
+EXPECTED_COLUMN_COUNT = len(EXPECTED_COLUMN_NAMES)
+
+
+def convert() -> None:
+    """Parse the markdown at the given URL and convert it to YAML."""
+    # Get the markdown
+    logging.info("Reading markdown at %s", RAW_URL)
```

Comment:
Proper capitalization:
```suggestion
    """Parse the Markdown at the given URL and convert it to YAML."""
    # Get the Markdown
    logging.info("Reading Markdown at %s", RAW_URL)
```